doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,69 @@
+iceoryx keeps things simple and offers basic Quality of Service (QoS) settings. The QoS
+settings are called options and can be used to optimize and tailor the communication.
+
+## `ServerOptions` and `ClientOptions`
+
+The two most important settings are:
+
+| Option                                 | Explanation                                                  |
+|----------------------------------------|--------------------------------------------------------------|
+| `ServerOptions::requestQueueCapacity`  | This enables servers to store n requests at the users side   |
+| `ClientOptions::responseQueueCapacity` | This enables clients to store n responses at the users side  |
+
+### Compatible policies
+
+The `ConsumerTooSlowPolicy` and `QueueFullPolicy` are used to control the behavior for full queues. Since some
+combinations would be contradicting, not all of them match to connect producer and consumer.
+
+| `ConsumerTooSlowPolicy`   | `QueueFullPolicy`     | Behavior                                   | Connection          |
+|---------------------------|-----------------------|--------------------------------------------|---------------------|
+| `WAIT_FOR_CONSUMER`       | `BLOCK_PRODUCER`      | Producer blocks and waits for subscriber   | :white_check_mark:  |
+| `WAIT_FOR_CONSUMER`       | `DISCARD_OLDEST_DATA` | Non-blocking producer                      | :white_check_mark:  |
+| `DISCARD_OLDEST_DATA`     | `DISCARD_OLDEST_DATA` | Non-blocking producer                      | :white_check_mark:  |
+| `DISCARD_OLDEST_DATA`     | `BLOCK_PRODUCER`      | Not compatible, no connection established  | :x:                 |
+
+## Server and client matching criteria
+
+Two criteria have to be fulfilled in order for a server and a client to be connected.
+
+1. Same `capro::ServiceDescription`
+2. Matching `ConsumerTooSlowPolicy` and `QueueFullPolicy` in `ServerOptions`/`ClientOptions`
+
+## `PublisherOptions` and `SubscriberOptions`
+
+The three most important settings are:
+
+| Option                              | Explanation                                                                                                                                       |
+|-------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| `PublisherOptions::historyCapacity` | This enables late-joining subscribers to request the n last samples                                                                               |
+| `SubscriberOptions::queueCapacity`  | Size of the subscriber queue where samples are stored at the user side                                                                            |
+| `SubscriberOptions::historyRequest` | The number of samples a late-joining subscriber will request from a publisher, should be equal or smaller than `historyCapacity` otherwise capped |
```

Comment:
@mossmaurice The behavior of today would maybe best be described with `The number of samples a late-joining subscriber will request from every offered publisher` Or? 
I would write here as it behaves today

Reply:
Added